ang/civicase/CaseOverview.html

```diff
@@ -3,16 +3,39 @@
     <div class="civicase__case-overview-container" civicase-custom-scrollbar scrollbar-config="{autoHide: false}">
       <div class="civicase__case-overview__flow">
         <div class="civicase__case-overview__flow-status">
+          <civicase-popover
+            class="civicase__case-overview__flow-status-settings"
+            popover-class="civicase__case-overview__popup">
+            <civicase-popover-toggle-button class="btn btn-link dropdown-toggle">
+              <i class="material-icons civicase__case-overview__flow-status__icon civicase__case-overview__flow-status__icon--settings">settings</i>
+              <i class="fa fa-caret-down civicase__case-overview__flow-status__icon civicase__case-overview__flow-status__icon--settings"></i>
+            </civicase-popover-toggle-button>
+            <civicase-popover-content>
+              <ul class="dropdown-menu" role="menu">
+                <li ng-repeat="(sid, status) in caseStatuses">
```

Comment:
please have a better name for the variable `sid`. I mean a more meaningful name.

Reply:
I'm removing the `sid` reference since it's not being used anyways.